code/API_definitions/CAMARA Connectivity Insights API.yaml

```diff
@@ -31,9 +31,12 @@
 
     
         Following diagram shows the interaction between different components
-        
-        documentation/API_documentation/sequenceDiagram.v1.png
-        
+        ![Sequence Diagram](https://github.com/camaraproject/ConnectivityInsights/blob/main/documentation/API_documentation/sequenceDiagram.v1.png)
```

Comment:
We should change this to Mermaid syntax to avoid having to recreate the diagram following changes n paths and resources.